apps/src/applab/commands.js

```diff
@@ -907,10 +913,15 @@ applabCommands.textInput = function(opts) {
   newInput.style.fontFamily = fontFamilyStyles[0];
   newInput.style.height = '30px';
   newInput.style.width = '200px';
-  elementUtils.setDefaultBorderStyles(newInput, {
-    forceDefaults: true,
-    textInput: true
-  });
+  if (experiments.isEnabled('applabThemes')) {
+    newInput.style.borderStyle = 'solid';
+    elementLibrary.applyCurrentTheme(newInput, Applab.activeScreen());
+  } else {
+    elementUtils.setDefaultBorderStyles(newInput, {
+      forceDefaults: true,
+      textInput: true
+    });
+  }
```

Comment:
Question: Why couldn't we push the experiment down into `elementLibrary.applyCurrentTheme`?  You've got a really clean divide between old and new behaviors here, but it seems like its also resulted in a lot of duplication (or at least proliferation of the experiment flag across many files).